Tests/scripts/destroy_instances.py

```diff
@@ -39,13 +40,16 @@ def main():
         except subprocess.CalledProcessError as exc:
             print(exc.output)
 
-        if os.path.isfile("./Tests/is_build_passed_{}.txt".format(env["Role"].replace(' ', ''))):
-            print(f'Destroying instance {env.get("Role", "Unknown role")}')
-            rminstance = aws_functions.destroy_instance(env["Region"], env["InstanceID"])
-            if aws_functions.isError(rminstance):
-                print_error(rminstance)
+        if time_to_live:
+            print(f'Skipping - Time to live was set to {time_to_live} minutes')
```

Comment:
this way you can avoid the `else` and extra indentation of the rest of the code.
```suggestion
        if time_to_live:
            print(f'Skipping - Time to live was set to {time_to_live} minutes')
            return
```